### PR TITLE
Junit4 Assignment

### PR DESCRIPTION
**Pull request by riyaknol11**

### CODE REVIEW

File: Junit4/src/main/java/com/knoldus/StringUtils.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package com.knoldus;
+
+
+class StringUtils{
+
+    public String reverse(String string){
```
Comment (anup0060) [inline Junit4/src/main/java/com/knoldus/StringUtils.java]: string - variable name , not good

File: Junit4/src/test/java/com/knoldus/StringUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    StringUtils stringUtils;
+
+    @Before
+    public void setup(){
+        stringUtils = new StringUtils();
+    }
+
+    // reverseStringTestMethod method to test the reverse method on given string
+    @Test
+    public void reverseStringTestMethodForNonEmptyString(){
```
Comment (anup0060) [inline Junit4/src/test/java/com/knoldus/StringUtilsTest.java]: testReverseString_givenExpectedInput -- like this, we need to have the name that reflects scenario as suffix.

**Review by anup0060 — CHANGES_REQUESTED** [review]

**Review by riyaknol11 — COMMENTED** [review]
Modified the naming conventions of methods and added required changes following the reviews

**Review by riyaknol11 — COMMENTED** [review]
Changes added in StringUtils class file, changes the string variable name

**Review by anup0060 — APPROVED** [review]

**Pull request merged**